MAINTAINERS.yml

```diff
@@ -4603,6 +4603,15 @@ West:
   labels:
     - "area: CMSIS-NN"
 
+"West project: cmsis_6":
+  status: maintained
+  collaborators:
+    - tomi-font
+    - henrikbrixandersen
+    - d3zd3z
+  labels:
+    - "area: CMSIS_6"
```

Comment:
I think at least 1 entry in `maintainers` is required for a project to be `maintained`.

`files` is mandatory, but can be an empty list.

Reply:
Indeed. Well, put myself, but anyone feel free to step up.